src/Entity/Post.php

```diff
@@ -204,4 +216,32 @@ public function setType(string $type): void
     {
         $this->type = $type;
     }
+
+    /**
+     * @return Collection|Tag[]
+     */
+    public function getTags(): Collection
+    {
+        return $this->tags;
+    }
+
+    /**
+     * @param iterable|Tag[] $tags
+     */
+    public function setTags(iterable $tags): void
```

Comment:
Is this setter so necessary? No type checks, and no opportunity to implement it. Mybe enogth addTag and removeTag methods?

Reply:
I guess that without this method, the symfony form does not mapping data correctly.